buildSrc/src/main/java/com/google/firebase/gradle/plugins/semver/GmavenCopier.kt

```diff
@@ -41,6 +41,11 @@ abstract class GmavenCopier : DefaultTask() {
         mavenHelper.getLatestReleasedVersion(),
         !aarAndroidFile.get()
       )
-    URL(gMavenPath).openStream().use { Files.copy(it, Paths.get(filePath.get())) }
+    try {
+      URL(gMavenPath).openStream().use { Files.copy(it, Paths.get(filePath.get())) }
+    } catch (_: java.io.FileNotFoundException) {
+      // Gmaven Artifact doesn't exist.
```

Comment:
You should be fine to replace these empty returns with `StopActionException` for more information.
```kotlin
throw StopActionException("GMaven artifact doesn't exist")
```